org.eclipse.scout.rt.shared/src/main/java/org/eclipse/scout/rt/shared/ui/UserAgentUtility.java

```diff
@@ -56,4 +56,25 @@ public static UserAgent getCurrentUserAgent() {
     }
   }
 
+  /**
```

Comment:
Is this method really necessary? It would also be possible to use the ClientRunContext instead?